### ISSUE TITLE
The igxGrid  keyboard navigation does not work when there are many collapsed groups at the bottom

### ISSUE THREAD

**Issue by PavlovVasil** [IgniteUI/igniteui-angular#4990]
## Description  
Navigating with Ctrl+ Arrow Up and then Ctrl + Arrow down does not work correctly when there is a large number of collapsed groups at the bottom of the grid.

 * igniteui-angular version: 7.3.x
 * browser: n/a

## Steps to reproduce  

1. Run the dev demos and open the GroupBy sample:  http://localhost:4200/gridGroupBy
2. Group by the "Address" column
3. Collapse the last 15 groups: everything below "Berkeley Gardens 12 Brewery" (it should be the last expanded group)
4. Select the "Consolidated Holdings" cell in that group
5. Press Ctrl + Arrow Up
6. Press Ctrl + Arrow Down

## Result  
Ctrl + Down does not scroll to the last visible cell - "Consolidated Holdings" in "Berkeley Gardens 12 Brewery". It scrolls to the bottom of the grid instead. Also, the focus is lost and the keyboard navigation stops working.

## Expected result  
Ctrl + Down should scroll to the last visible cell

## Attachments  

![GroupByKeyboardNav](https://user-images.githubusercontent.com/18643062/58954340-97562380-87a1-11e9-8527-428e46b28a16.gif)
